### ISSUE TITLE
Tribler will not show anything in Results, Downloads, Channels content, etc. screens.

### ISSUE THREAD

**Issue by CedarD** [Tribler/tribler#1101]
I currently am running Windows 8.1 and was trying out Tribbler. Its performance was fine for a while, but leaving it open, eventually the application became sluggish and eventually froze. Trying to close/kill the application, first by the dialogue that Windows pops up, and after that failed, by using the Task Manager to attempt to kill the process itself. At this point, the whole OS became unresponsive for a short bit before blue screening with a DPC_WATCHDOG_VIOLATION error.

Upon rebooting, Windows forced an automatic repair which failed and subsequently I opted to revert to an earlier state. After that was done, the machine booted up fine and all the applications I've used thus far were working properly, with the exception of Tribler.

When I try to search for anything, the little progress bar on the search button goes to completion, but the Results pane always says "Loading, please wait." In Downloads, pane it also says "Loading, please wait." Trying to reopen a download Tribler was doing prior to this results in a message: "Duplicate download/You are already downloading this torrent, see the Downloads section."

Tribler appears to be downloading something, or perhaps it's just normal background traffic for the application, but it says I'm getting some download traffic, no upload traffic, "Port is working" and there is some information when I hover over the "Person connected" icon. In the Channels pane, there is a large list of "channels" so as far as I can tell, at least that part is working. Though clicking to view the contents of a channel displays a "Loading, please wait." screen.

I have already tried uninstalling and reinstalling Tribler and this did not help.

**Comment by whirm** [comment]
@CedarD any update on this?

**Comment by CedarD** [comment]
Nothing so far. No DPC violations "caused" by Tribbler since then, however
occasionally it has become unresponsive but has been killable a few times.
Results still show when I search since I reinstalled completely &
regcleaned. I will update if the problem reoccurs. A few other applications
have been blamed here and there for DPC violations by windbg since I opened
this issue. (Warframe, Chrome, etc.) Not certain what's the root cause,
it's probably Chrome or possibly one of its plugins I'll bet since Chrome's
been open during each DPC violation incident.
